rfcs/text/0001-rfc-process.md

```diff
@@ -70,103 +84,111 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
 ## Detailed Design
 
-### Proposing RFCs
+### Drafting RFCs
 
 #### **Authors**:
 
-*Before proposing, always engage the DCP community to build consensus and assess whether your proposal sparks interest and addresses a need. You may even discover potential collaborators.*
+There are three types of RFCs that may be proposed: ***Software***, ***Governance***, and ***Informational***. Software RFCs are expected to be linked to an Objective that is prioritized in the DCP Product Roadmap, tracked by linking directly to an existing ticket in the DCP Zenhub board (usually labeled as a Spike). Informational and Governance RFCs need not directly reference an open ticket in the DCP Zenhub board.
```

Comment:
1. Should there be a definition for  [**Informational**](https://github.com/HumanCellAtlas/dcp-community/issues/30)?
2. Should its approval be _approval to publish_ per my [comments](https://github.com/HumanCellAtlas/dcp-community/pull/70#issue-257148669) in the _prototype_ Informational RFC?
3. _Software RFCs are expected to be linked_ -> _Software RFCs SHOULD be linked_
3. What does _linking_ mean in this context? For example, is this [Connecting Pull Requests to GitHub Issues](https://help.zenhub.com/support/solutions/articles/43000010350-connecting-pull-requests-to-github-issues), or adding a reference to the blocked roadmap objective in the top-level summary comment of the PR?